build.gradle.kts

```diff
@@ -27,7 +27,7 @@ description = projectDescription
 object Versions {
     const val androidTools = "7.2.1"
     const val junit = "4.13.2"
-    const val ktlint = "0.46.1"
+    const val ktlint = "0.47.0"
```

Comment:
Let's go to [0.47.1](https://github.com/pinterest/ktlint/releases/tag/0.47.1)!
I've gotten into the habit of waiting a few days after each ktlint release because there's always a flurry of error reports and fast fixes. Seems like some important ones to pick up.